src/main/java/com/openlattice/integrations/pennington/ReferenceCharges.java

```diff
@@ -0,0 +1,552 @@
+
+    private static final String VIOLENT_FQN    = "ol.violent";
+    private static final String DMF_STEP_2_FQN = "ol.dmfstep2indicator";
+    private static final String DMF_STEP_4_FQN = "ol.dmfstep4indicator";
+    private static final String BHE_FQN        = "ol.bheindicator";
+    private static final String BRE_FQN        = "ol.breindicator";
+
+    private static final String MINN_COURT_CHARGES    = "minnehahacourtcharges";
+    private static final String MINN_ARREST_CHARGES   = "minnehahaarrestcharges";
+    private static final String PENN_COURT_CHARGES    = "penningtoncourtcharges";
+    private static final String PENN_ARREST_CHARGES   = "penningtonarrestcharges";
+    private static final String SHELBY_COURT_CHARGES  = "Shelby County_publicsafety_courtchargelist";
+    private static final String SHELBY_ARREST_CHARGES = "Shelby County_publicsafety_arrestchargelist";
+
+    private static final String LINCOLN_COURT_CHARGES  = "lincolncountysd_publicsafety_courtchargelist";
+    private static final String LINCOLN_ARREST_CHARGES = "lincolncountysd_publicsafety_arrestchargelist";
+
+    private static final String PTCM_ARREST_CHARGES = "OpenLattice PCM Demo Org_publicsafety_arrestchargelist";
+    private static final String PTCM_COURT_CHARGES  = "OpenLattice PCM Demo Org_publicsafety_courtchargelist";
+
+    private static final String SD_APPEARS_IN      = "southdakotaappearsin";
+    private static final String COUNTIES           = "southdakotacounties";
+    private static final String COURTHOUSES        = "southdakotacourthouses";
+    private static final String COURTROOMS         = "southdakotacourtrooms";
+    private static final String AGENCIES           = "southdakotaagencies";
+    private static final String REMINDER_TEMPLATES = "southdakotaremindertemplates";
+    private static final String CONTACT_INFO       = "southdakotacontactinformation";
+    private static final String CONTACT_INFO_GIVEN = "southdakotacontactinfogiven";
+
+    private static String getChargeId( Map<String, Object> row ) {
+        String statute = Parsers.getAsString( row.get( ChargeConstants.STATUTE ) );
+        String description = Parsers.getAsString( row.get( ChargeConstants.DESCRIPTION ) );
+        if ( statute == null ) {
+            statute = "";
+        }
+        if ( description == null ) {
+            description = "";
+        }
+
+        return statute + "|" + description;
+    }
+
+    private static boolean getIsViolent( Row row ) {
+        return Parsers.parseBoolean( row.getAs("voilent") );
+    }
+
+    private static boolean getBooleanValue( Object value ) {
+        String strValue = Parsers.getAsString( value );
+        if ( StringUtils.isNotBlank( strValue ) ) {
+            return strValue.trim().toLowerCase().equals( "true" );
+        }
+
+        return false;
+    }
+
+    public static String getValue( Row row, String header ) {
+        String val = Parsers.getAsString( row.getAs( header ) );
```

Comment:
no need for anything below this -- this is all taken care of by parsers. This function should just be `return Parsers.getAsString( row.getAs( header ) );`